build/ci/create_demo_pr.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -xe
+set +o pipefail
+
+sed -i "s+$operator_image_in_branch+$operator_image_for_test+g" $csv_file
+
+# install gh command
+curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo gpg --dearmor -o /usr/share/keyrings/githubcli-archive-keyring.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+sudo apt-get update
+sudo apt-get install gh
```

Comment:
shouldn't this be part of the prepare/setup env job (similiar to `yq`)?

Reply:
can [this](https://stackoverflow.com/a/57958803) help?

Reply:
done